src/shared/components/ResourceForm/defaultResourcePayloads.ts

```diff
@@ -1,66 +1,50 @@
 export const DEFAULT_RESOURCE = {
-  '@context': {}, // optional
   '@id': '', // optional
   '@type': [], // optional
 };
 
 export const DISK_STORAGE = {
-  '@context': {}, // optional
   '@id': '', // optional
   '@type': 'DiskStorage',
   default: false,
   volume: '',
-  readPermission: [], // optional
-  writePermission: [], // optional
 };
 
 export const REMOTE_STORAGE = {
-  '@context': {}, // optional
   '@id': '', // optional
   '@type': 'RemoteDiskStorage',
   default: false,
-  endpoint: '',
-  credentials: '', // optional
-  readPermission: [], // optional
-  writePermission: [], // optional
+  folder: '',
 };
 
 export const S3_STORAGE = {
```

Comment:
s3 storage is not supported any more. Can we remove this all together (should also remove from the drop down).